js/scripts.js

```diff
@@ -0,0 +1,19 @@
+function Button(text) {
+	this.text = text || 'Hello!';
+}
+
+Button.prototype = {
+	create: function() {
+		var self = this;
+		this.$element = $('<button>');
+		this.$element.text(this.text);
+		this.$element.click(function(){
```

Comment:
use method `on` instead of `click`

add one more event - when somebody focus on button show message above button "Hej! Stop doing this" ;)